target/linux/ath79/dts/ar9344_mikrotik_routerboard-sxt-5nd-r2.dts

```diff
@@ -6,3 +6,11 @@
 	compatible = "mikrotik,routerboard-sxt-5nd-r2", "qca,ar9344";
 	model = "MikroTik RouterBOARD SXT 5nD r2 (SXT Lite5)";
 };
+
+&routerboot {
+	partition@10000 {
+		label = "bootloader2";
+		reg = <0x10000 0x10000>;
+		read-only;
+	};
+};
```

Comment:
Please don't do this. This makes no sense, and will be a maintenance nightmare. If the partition tables differ between devices, each DTS should have the full table I think. @adschm opinion?

Reply:
I don't have a problem with splitting in general. One can improve the situation by adding comments to the possibly "incomplete" setup in the DTSI. I'd be more cautious with the routerboot situation, though, since this harder to grasp than the simple list of partitions you typically have.

So, no, I don't reject this generally. However, with the answer on the other question with respect to the size, this option is dead anyway as the size requirement will also require to have two full definitions.

Reply:
Inevitably the NOR variant will be added, which will further clobber the visibility if kept like this. For readability full flash table would be preferable.

Reply:
OK, this was fixed, as each device got its partition table.